backend/infrahub/webhook/models.py

```diff
@@ -95,7 +96,13 @@ async def _prepare_payload(self) -> None:
         default_branch = repo.default_branch
         commit = repo.get_commit_value(branch_name=default_branch)
 
-        self._payload = await repo.execute_python_transform(
+        timeout = 10
```

Comment:
It might be good to have a `DEFAULT_TRANSFORM_TIMEOUT` within a `constant.py` file (or at top of this file). It could then be reused within below tests